pkg/provider/azure_loadbalancer.go

```diff
@@ -3348,6 +3453,20 @@ func equalSubResource(s *network.SubResource, t *network.SubResource) bool {
 	return strings.EqualFold(pointer.StringDeref(s.ID, ""), pointer.StringDeref(t.ID, ""))
 }
 
+func equalSubResourceForIPv6Only(s *network.SubResource, t *network.SubResource, isIPv6Only bool) bool {
```

Comment:
In single stack, there will be no v6Suffix and in dual stack, there will always be v6Suffix. Do we still need this?

Reply:
Thank you for reminding! Removed.